tests/Feature/AbstractRequestTest.php

```diff
@@ -128,8 +132,9 @@ public function testCacheHitHasAccessToOriginalLog(): void
         self::assertSame(1, $tapper->getCountLike('POST', '/awesome/'));
         // Result was in cache
         self::assertTrue($secondRequest->isFromCache());
-        // Second request can retrieve log from original time
+        // Second request can retrieve log from original time with original contents
```

Comment:
Lines 135 and 137 should have shipped with 1.1.1, I apparently didn't commit them in that branch 😳